### PR TITLE
Add parameters to accept for StaffFireAction

### PR DESCRIPTION
**Pull request by csm10495**
Without this, plugins can call stafffire, but they can't set whom to fire.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Could you bump the plug-in api version

File: src/openrct2/actions/StaffFireAction.cpp [hunk 1]
```diff
@@ -18,6 +18,11 @@ StaffFireAction::StaffFireAction(EntityId spriteId)
 {
 }
 
+void StaffFireAction::AcceptParameters(GameActionParameterVisitor& visitor)
+{
+    visitor.Visit("spriteId", _spriteId);
```
Comment (IntelOrca) [inline src/openrct2/actions/StaffFireAction.cpp:23]:
```suggestion
    visitor.Visit("id", _spriteId);
```

**Review by IntelOrca — CHANGES_REQUESTED** [review]

**Comment by csm10495** [comment]
Updated. Thanks folks.

**Review by duncanspumpkin — APPROVED** [review]

**Review by tupaschoal — COMMENTED** [review]
Ok to merge this @IntelOrca ?

**Review by IntelOrca — APPROVED** [review]

**Pull request merged**